install_config/storage_examples/index.adoc

```diff
@@ -10,3 +10,4 @@ The following sections provide detailed conprehensive instructions on setting up
 
 - link:../../install_config/storage_examples/shared_storage.html[Sharing an NFS PV Across Two Pods]
 - link:../../install_config/storage_examples/ceph_examples/ceph_example.html[Ceph-RBD Block Storage Volume]
+- link:../../install_config/storage_examples/shared_storage.html[Mounting Persistent Volumes to Privileged Pods]
```

Comment:
I would abbreviate the Persistent Volumes to PV here for space along the LHS concerns